packages/client/src/pages/forum/components/ListTopic/index.tsx

```diff
@@ -1,206 +1,24 @@
-      date: new Date(),
-    },
-    {
-      id: 12,
-      title: 'Title topic12',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 13,
-      title: 'Title topic13',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 14,
-      title: 'Title topic14',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 15,
-      title: 'Title topic15',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 16,
-      title: 'Title topic16',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 17,
-      title: 'Title topic17',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 18,
-      title: 'Title topic18',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 19,
-      title: 'Title topic19',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 20,
-      title: 'Title topic20',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 21,
-      title: 'Title topic21',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 22,
-      title: 'Title topic22',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 23,
-      title: 'Title topic23',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 24,
-      title: 'Title topic24',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 25,
-      title: 'Title topic25',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 26,
-      title: 'Title topic26',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 27,
-      title: 'Title topic27',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 28,
-      title: 'Title topic28',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 29,
-      title: 'Title topic29',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-    {
-      id: 30,
-      title: 'Title topic30',
-      shortDescription: 'short description topick',
-      date: new Date(),
-    },
-  ];
+  const [topics, setTopics] = useState<ChatsEntity | []>([])
 
   const [currantPage, setCurrantPage] = useState(1);
 
   const changeCurantPage = (page: number) => {
     setCurrantPage(page);
   };
 
+  useEffect(() => {
+    getChats().then(data => setTopics(data))
+  }, [])
```

Comment:
_Не критично, но лучше исправить_

В это место хорошо вписывается фронтовый лог об ошибке запроса
```suggestion
  useEffect(() => {
    getChats().then(data => setTopics(data)).catch(e => <тут console.error или логирование>)
  }, [])
```